index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <main id="main">
+          <h1 id="title">
+            Dr Norman Borlag
+          </h1>
+          <p>The man who saved a billion lives</p>
+          <div class="img-div">
+            <img id="image" src="/image/norman.jpg" alt="image" srcset="">
+            <div id="img-caption">
+              <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
```

Comment:
I think it would be better to use class attributes throughout your code in order to apply the HTML & CSS best practices. Check [Best practices curriculum](”https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md”) to refresh your knowledge of HTML & CSS best practices.
Thanks 😀